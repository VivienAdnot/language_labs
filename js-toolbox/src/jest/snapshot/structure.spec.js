const structureCommon = require('./structureCommon');

it('generates ok', () => {
  const struct = structureCommon({ name: 'foo', objective: 'REACH' });
  expect(struct).toMatchSnapshot();
});
