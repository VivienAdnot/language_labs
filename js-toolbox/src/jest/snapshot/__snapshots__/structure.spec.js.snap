// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates ok 1`] = `
Object {
  "campaigns": Array [
    Object {
      "name": "foo",
      "objective": "REACH",
      "special_ad_category": "NONE",
      "status": "PAUSED",
    },
  ],
  "version": "1.0",
}
`;
